### DISCUSSION TITLE
Request to assign a task

### DISCUSSION THREAD

**Discussion by FussuChalice** [sircharlo/meeting-media-manager#2258] [General]
Hello dear developers,
I would like to offer my contribution to this project and work on the task in code.

I have experience in Electron.js, Js, Node.js, React.js and other technologies, and am ready to contribute to the project. If you have additional recommendations or guidance regarding this task, please provide them.

Thank you for your attention and I look forward to working on this challenge.

Best wishes,
Max

**Comment by mtdvlpr** [comment]
Hi @FussuChalice, thank you for your willing spirit! I'm currently busy with a big refactor (see https://github.com/mtdvlpr/meeting-media-manager/tree/refactor/next). If you want, you could get familiar with that codebase and help complete the refactor. All current M3 features should be present and working in the refactor. I have released an initial beta version which you could test and report/fix any bugs you find.

**Reply by mtdvlpr** [reply]
See https://github.com/mtdvlpr/meeting-media-manager/blob/refactor/next/CONTRIBUTING.md for instructions to get the refactor running locally.

**Comment by FussuChalice** [comment]
Thanks!
